scss/_modal.scss

```diff
@@ -48,11 +48,11 @@ $component-name: modal;
     padding-top: 1.875rem;
 
     > :not(:first-child) {
-      margin-right: 0.3125rem;
+      margin-right: .3125rem;
```

Comment:
Why do you add the `0` before at some place and here, you remove it ?

Reply:
Well, commits which removes zeros are stylelint auto-fix commits, others are format on save of my IDE, it looks like it doesn't take this stylelint config (gonna investigate on this)

Reply:
Thanks ;)